src/plugins/auto/tests/unit/release_helper_test.cpp

```diff
@@ -34,6 +34,112 @@ class AutoReleaseHelperTest : public tests::AutoTest, public ::testing::TestWith
+                compile_model(::testing::Matcher<const std::shared_ptr<const ov::Model>&>(_),
+                              ::testing::Matcher<const std::string&>(StrEq(ov::test::utils::DEVICE_GPU)),
+                              _))
+            .WillByDefault(InvokeWithoutArgs([this]() {
+                std::this_thread::sleep_for(std::chrono::milliseconds(2000));
+                return mockExeNetworkActual;
+            }));
+    } else {
+        ON_CALL(*core,
+                compile_model(::testing::Matcher<const std::shared_ptr<const ov::Model>&>(_),
+                              ::testing::Matcher<const std::string&>(StrEq(ov::test::utils::DEVICE_GPU)),
+                              _))
+            .WillByDefault(InvokeWithoutArgs([this]() {
+                return mockExeNetworkActual;
+            }));
+    }
+    if (cpuSleep) {
+        ON_CALL(*core,
+                compile_model(::testing::Matcher<const std::shared_ptr<const ov::Model>&>(_),
+                              ::testing::Matcher<const std::string&>(StrEq(ov::test::utils::DEVICE_CPU)),
+                              _))
+            .WillByDefault(InvokeWithoutArgs([this]() {
+                std::this_thread::sleep_for(std::chrono::milliseconds(2000));
+                return mockExeNetwork;
+            }));
+    } else {
+        ON_CALL(*core,
+                compile_model(::testing::Matcher<const std::shared_ptr<const ov::Model>&>(_),
+                              ::testing::Matcher<const std::string&>(StrEq(ov::test::utils::DEVICE_CPU)),
+                              _))
+            .WillByDefault(InvokeWithoutArgs([this]() {
+                return mockExeNetwork;
+            }));
+    }
+    metaDevices = {{ov::test::utils::DEVICE_CPU, {}, -1}, {ov::test::utils::DEVICE_GPU, {}, -1}};
+    DeviceInformation devInfo;
+    ON_CALL(*plugin, parse_meta_devices(_, _)).WillByDefault(Return(metaDevices));
+    ON_CALL(*plugin, get_valid_device)
+        .WillByDefault([](const std::vector<DeviceInformation>& metaDevices, const std::string& netPrecision) {
+            std::list<DeviceInformation> devices(metaDevices.begin(), metaDevices.end());
+            return devices;
+        });
+    ON_CALL(*plugin, select_device(Property(&std::vector<DeviceInformation>::size, Eq(2)), _, _))
+        .WillByDefault(Return(metaDevices[1]));
+    ON_CALL(*plugin, select_device(Property(&std::vector<DeviceInformation>::size, Eq(1)), _, _))
+        .WillByDefault(Return(metaDevices[0]));
+    config.insert(ov::device::priorities(ov::test::utils::DEVICE_CPU + std::string(",") + ov::test::utils::DEVICE_GPU));
+    std::shared_ptr<ov::ICompiledModel> exeNetwork;
+    std::string result;
+    ASSERT_NO_THROW(exeNetwork = plugin->compile_model(model, config));
+    if (actSleep) {
+        if (!cpuSleep) {
+            ASSERT_NO_THROW(result = exeNetwork->get_property(ov::model_name.name()).as<std::string>());
+            EXPECT_EQ(result, modelNameCpu);
```

Comment:
let's iterate all supported property not only model_name?

Reply:
you mean we should iterate all of the supported property without exception regardless of whether CPU_HELP has been reset? or we should have some expectations for the returned property value like the `ov::model_name`?

Reply:
yes, I mean other properties like optimal inter request etc.. are there cases to cover this bug?

Reply:
Updated.